spec/rails_helper.rb

```diff
@@ -62,3 +62,9 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+  config.hook_into :webmock
+  config.configure_rspec_metadata!
```

Comment:
Remember to filter out your API key in the future! But nice work getting VCR up and running for this assessment!